### ISSUE TITLE
figure out a way to detect ohloh rank and swap in transparent image.

### ISSUE THREAD

**Issue by myusuf3** [pyconca/2012-web#57]

**Comment by merwok** [comment]
I don’t see the link between ohloh rank and pictures, could you elaborate?

**Comment by myusuf3** [comment]
so on the pyconca site for  speakers who had ohloh accounts we pulled in an image associated with their rank. That image has a white background which makes it look ugly on the site, so I had to used photoshop to make the backgrounds   transparent so they looked good. 

We need a way to tie speaker's name to ohloh and process their ranking image with PIL or something to make the background transparent.

**Comment by dianaclarke** [comment]
Yes, hardcoded rank is not ideal. That said, I'm not sure that we'll be using the same application this year for PyCon Canada. I think Taavi is leading the web-dev this year; co-ordinate before implementing. Cheers!
